django_app/report_a_suspected_breach/templates/report_a_suspected_breach/form_steps/end_user_added.html

```diff
@@ -1,13 +1,36 @@
 {% extends "report_a_suspected_breach/base_form_step.html" %}
 {% load crispy_forms_tags crispy_forms_gds %}
 {% load get_wizard_step_url %}
+{% load countries %}
 
-{% block form_content %}
+{% block column_content %}
     <h1 class="govuk-heading-l">
         You've added {{ request.session.end_users.keys|length }} end-user{% if request.session.end_users.keys|length > 1 %}s{% endif %}
     </h1>
     {% for id, end_user in request.session.end_users.items %}
-        <h3 class="govuk-heading-m">End-user {{ forloop.counter }}</h3>
+        <div class="action-wrapper">
+            <div>
+                <h3 class="govuk-heading-m" style="float:left;">End-user {{ forloop.counter }} </h3>
+                <ul class="govuk-summary-card__actions">
+                    <dd class="govuk-summary-list__action govuk-!-font-weight-regular">
+
+                        <a class="govuk-link" style="font-size: 19px; text-align:right"
+                           href="{% url 'report_a_suspected_breach:about_the_end_user' end_user_uuid=id %}">Change<span
+                            class="govuk-visually-hidden"> End-user {{forloop.counter}} details </span></a>
+                    </dd>
+                    <dd class="govuk-summary-list__action">
+                        <form id="delete_end_user_form_{{ forloop.counter }}" method="post" enctype='multipart/form-data' novalidate
+                              action="{% url 'report_a_suspected_breach:delete_end_user' %}?end_user_uuid={{ id }}">
+                            {% csrf_token %}
```

Comment:
instead of using URL parameters, as it's a post request it makes more sense to include the `id` as a hidden input:

```suggestion
 <input type="hidden" name="end_user_uuid" value="{{ id }}">
                            {% csrf_token %}
```